quarkchain/evm/specials.py

```diff
@@ -237,16 +237,32 @@ def proc_current_mnt_id(ext, msg):
 def proc_transfer_mnt(ext, msg):
     from quarkchain.evm.messages import apply_msg
 
-    # Data must be >= 96 bytes
-    if msg.data.size < 96:
-        return 0, 0, []
-    gascost = 3
-    if msg.gas < gascost:
+    # Data must be >= 96 bytes and static call not allowed
+    if msg.data.size < 96 or msg.static:
```

Comment:
I think it is right to check msg.static, but do you think we need to accept an input parameter to issue a static call?  It looks like the static call should be used for all transfer/send in solidity rather than supplying gas with GSTIPENT

Reply:
Oh, I double-check the code again, and actually, to call STATICCALL, the value must be 0 in EVM.  This means we should still reject any call with msg.static = true.